src/modules/clock.cpp

```diff
@@ -196,6 +196,7 @@ template <>
 struct fmt::formatter<waybar_time> : fmt::formatter<std::tm> {
   template <typename FormatContext>
   auto format(const waybar_time& t, FormatContext& ctx) {
+    basic_memory_buffer<char> tm_format;
```

Comment:
This breaks clock formatting when a timezone is specified.

I'd try following, but let me think more on that...
```suggestion
#if FMT_VERSION >= 80000
    auto& tm_format = specs;
#endif
```